apps/service_providers/llm_service/helper.py

```diff
@@ -57,6 +53,10 @@ def custom_parse_ai_message(message) -> list[AgentAction] | AgentFinish:
         else:
             tool_input = _tool_input
 
+        # Skip malformed built-in tool calls that originate from Anthropic models
```

Comment:
Would this only happen sometimes?

Reply:
It happens every time Anthropic does tool calls for internal tools like 'web search'. I haven't determined if it's a Langchain artefact or not because it's not included in their example response in the docs: https://docs.anthropic.com/en/docs/agents-and-tools/tool-use/web-search-tool#response